lib/net.c

```diff
@@ -64,6 +64,13 @@ get_net_conf(void)
 	return &config;
 }
 
+uint16_t
+lcore_to_queue(unsigned int lcore)
```

Comment:
This function needs two more parameters to know which kind of queue to map (i.e. tx or rx), and from which interface (i.e. front, and back).